.github/workflows/publish.yml

```
@@ -0,0 +1,30 @@
+name: Publish
+on:
+  push:
+    branches: master
+    paths: lib/rubocop/rspec/version.rb
+jobs:
+  publish:
+    name: Publish to RubyGems
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: ruby
+      - uses: rubygems/release-gem@v1
+      - name: Replace version in README
+        run: |
+          sed -i "s/version:.*$/version: ~/g" docs/antora.yml
```

Let’s remove the `g` option and use single quotes:
```suggestion
          sed -i 's/version:.*$/version: ~/' docs/antora.yml
```